### PR TITLE
Features/edu benefits application instructions

### PR DESCRIPTION
**Pull request by FernandoBoza**
## Description
Spacing between OMBInfo and SaveInProgress component added. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Definition of done
- [x] Spacing between OMBInfo and SaveInProgress component added

### CODE REVIEW

**Comment by zurbergram** [comment]
Looks like a bad merge occurred

**Comment by FernandoBoza** [comment]
> Looks like a bad merge occurred

yes sorry, reverting to previous commit before the mess.

**Review by khansahammami — APPROVED** [review]

File: src/platform/forms-system/src/js/components/ReviewBoxField.jsx
Comment (bkjohnson) [inline src/platform/forms-system/src/js/components/ReviewBoxField.jsx:24]: I believe @nathanielbuck had something similar on his branch that I looked at earlier. I'm going to hold off on reviewing this since I suspect it may be a mistake.

Comment (nathanielbuck) [reply src/platform/forms-system/src/js/components/ReviewBoxField.jsx:24]: Yeah, @FernandoBoza had copied some in-progress code of mine to use for testing purposes since we're working on similar stuff in parallel. The version in my PR is a little more updated, so feel free to leave any comments or suggestions for improvement there: https://github.com/department-of-veterans-affairs/vets-website/pull/18099

File: src/applications/my-education-benefits/components/EmailViewField.jsx
Comment (nathanielbuck) [inline src/applications/my-education-benefits/components/EmailViewField.jsx:4]: formData may not be defined here if the email field isn't pre-populated for a veteran (unlikely, but possible if whatever backend services don't have it). So, I'd suggest returning an empty string here (or just returning if a string isn't expected).

**Review by khansahammami — APPROVED** [review]

**Comment by FernandoBoza** [comment]
@khansahammami

**Review by khansahammami — APPROVED** [review]

**Review by khansahammami — APPROVED** [review]

**Review by khansahammami — APPROVED** [review]

**Comment by FernandoBoza** [comment]
@bkjohnson can you please review PR when possible, thank you

**Review by bkjohnson — COMMENTED** [review]
I expect that this may be discussed as part of an upcoming meeting, so I'll hold off on reviewing until afterwards.

**Review by bkjohnson — CHANGES_REQUESTED** [review]
This PR has 500+ lines of changes and a description of:

> Spacing between OMBInfo and SaveInProgress component added.

If I filter the files to only show me the ones that I'm a codeowner of I'm left with one file, and it doesn't have any references to the `OMBInfo` or `SaveInProgress` components. As a potential reviewer, this puts me in a position where I don't really know what the goal with the PR is or what to expect due to a description that doesn't seem to match the changes.

Here is some documentation that you can use to get this PR into a review-ready state.

- [What we look for when we review your code](https://depo-platform-documentation.scrollhelp.site/developer-docs/Submitting-pull-requests-for-approval.655032351.html#Submittingpullrequestsforapproval-Whatwelookforwhenwereviewyourcode)
- [Clearly summarize the change in the PR title and description](https://depo-platform-documentation.scrollhelp.site/developer-docs/Pull-request-best-practices.688259145.html#Pullrequestbestpractices-ClearlysummarizethechangeinthePRtitleanddescription)

**Pull request closed without merging**
